src/components/RecordPicker.js

```diff
@@ -281,7 +281,7 @@ function RecordPicker(props) {
 			.sort((a, b) => {
 				debugger
 				// Handle quantitative values with parseInt
-				if (sortField.type === 'quantitative' || sortField.name === 'pid')
+				if (sortField === 'quantitative' || sortField === 'pid')
```

Comment:
This doesn’t look quite right. It’s the type column in codebook.csv that determines the variable type. So shouldn’t it be sortField.type? Also, I don’t think we should hard code pid in there, and pid is anyway quantitative so it shouldn’t make any difference.

Reply:
Hey - I did not realise what the type === quantitative was doing in this function. It looks like all values that are type quantitative are expected to be numbers, is that right?